tests/acceptance/PageFormCest.php

```diff
@@ -0,0 +1,232 @@
+<?php
+/**
+ * Tests for ConvertKit Forms on WordPress Pages.
+ * 
+ * @since 	1.0.0
+ */
+class PageFormCest
+{
+	/**
+	 * Run common actions before running the test functions in this class.
+	 * 
+	 * @since 	1.0.0
+	 * 
+	 * @param 	AcceptanceTester 	$I 	Tester
+	 */
+    public function _before(AcceptanceTester $I)
+    {
+    	$I->activateConvertKitPlugin($I);
+    	$I->setupConvertKitPlugin($I);
+    }
+
+    /**
+	 * Test that the 'Default' option for the Default Form setting in the Plugin Settings works when
+	 * creating and viewing a new WordPress Page, and there is no Default Form specified in the Plugin
+	 * settings.
+	 * 
+	 * @since 	1.0.0
+	 * 
+	 * @param 	AcceptanceTester 	$I 	Tester
+	 */
+    public function testAddNewPageUsingDefaultFormWithNoDefaultFormSpecifiedInPlugin(AcceptanceTester $I)
+    {
+    	// Navigate to Pages > Add New
+        $I->amOnAdminPage('post-new.php?post_type=page');
+
+        // Check that no PHP warnings or notices were output.
+    	$I->checkNoWarningsAndNoticesOnScreen($I);
```

Comment:
I'm curious why you're running this here when no actions have been performed and why it's not present in all tests.

Reply:
I may be misunderstanding the questions/scope here:

> I'm curious why you're running this here when no actions have been performed
I believe loading the Pages > Add New screen in the WordPress Administration is an action, and we're checking for xdebug errors and the .php-error body class because 1.9.5.2 throws a *lot* of these errors when making the subsequent call to `$I->checkNoWarningsAndNoticesOnScreen($I);`

> why it's not present in all tests.
As an example, the ActivateDeactivatePluginCest tests call helper functions `activateConvertKitPlugin` and `deactivateConvertKitPlugin`.  These do include the `$I->checkNoWarningsAndNoticesOnScreen($I);` check.
If you have specific test examples where you feel the chain of assertions never makes a call to `$I->checkNoWarningsAndNoticesOnScreen($I);`, add a comment to them so I can review.

Reply:
Thanks for the explanation!

> 1.9.5.2 throws a lot of these errors when making the subsequent call to $I->checkNoWarningsAndNoticesOnScreen($I);

This still peaks my curiosity since why would simply loading a screen be an action that could potentially causing errors?

There weren't any tests that were conspicuously missing them, just some that made that assertion twice and others that only did it once.

I'm new to Codeception, is `codeception.com/docs` the best resource to get started?

Reply:
Screenshot attached showing PHP errors emitting from the 1.9.5.2 ConvertKit Plugin when installed and activated for the first time and the user then performs the action of visiting Pages > Add New

<img width="1792" alt="Screenshot 2021-11-12 at 09 24 39" src="https://user-images.githubusercontent.com/1462305/141491349-ce1d5621-e691-4e8b-bf7b-ea4f268b2f9f.png">

Hope that explains why we test for errors on every screen.

Reply:
Cool, thanks for the explanation!